odl/discr/lp_discr.py

```diff
@@ -34,9 +34,7 @@
 from odl.discr.partition import RectPartition, uniform_partition_fromintv
 from odl.set.sets import RealNumbers, ComplexNumbers
 from odl.set.domain import IntervalProd
-from odl.space.cu_ntuples import CUDA_AVAILABLE, CudaFn
-from odl.space.fspace import FunctionSpace
-from odl.space.ntuples import Fn
+from odl.space import FunctionSpace, FN_IMPLS
```

Comment:
We should have some sort of naming convention for such global sequences, I'd be good with `<XYZ>_IMPLS`. Should be applied across the board then.